crypto/src/test/java/org/web3j/crypto/TransactionEncoderTest.java

```diff
@@ -93,7 +93,7 @@ public void testEip1559Transaction() {
                 TransactionEncoder.signMessage(
                         createEip1559RawTransaction(), (byte) 1, credentials),
```

Comment:
I think passing 1 as chainId here is a bit weird and the chainID coming from createEip1559RawTransaction is 1559 - this feels really odd

Reply:
Yes thanks I updated the test to have a consistent usecase